resources/js/src/app/components/common/LazyImg.vue

```diff
@@ -1,23 +1,50 @@
 <template>
-    <img v-if="!isBackgroundImage" :data-src="imageUrl">
-    <div v-else :data-background-image="imageUrl">
+    <picture v-if="!isBackgroundImage" :data-iesrc="fallbackUrl || imageUrl" :data-picture-class="sizingClass">
+        <source :srcset="imageUrl" :type="mimeType">
+        <noscript><img :src="fallbackUrl || imageUrl"></noscript>
+    </picture>
+    <div v-else :data-background-image="backgroundSource" :class="sizingClass">
         <slot></slot>
     </div>
 </template>
 
 <script>
-import lozad from "lozad";
+import lozad from "../../plugins/lozad";
+import { detectWebP } from "../../helper/featureDetect";
 
 export default {
     props: {
-        imageUrl: String,
-        isBackgroundImage: Boolean
+        imageUrl: {
+            type: String,
+            required: true
+        },
+        fallbackUrl: String,
+        isBackgroundImage: Boolean,
+        sizingClass: String
```

Comment:
Können wir diese Property noch umbenennen in `pictureClass`? Das beschreibt nach außen eher, wofür die property verwendet wird.